src/components/common/Wrapper.tsx

```diff
@@ -44,13 +44,13 @@ export const Space = styled.div<{
   css?: ReturnType<typeof css>;
 }>`
   height: ${({ height }) =>
-    height && typeof height === 'number'
+    typeof height === 'number'
       ? `${height}px`
       : height && height in wowSpace
         ? wowSpace[height]
         : ''};
```

Comment:
p5; 
지금 삼항 연산자가 복잡하게 얽혀있어서 아래와 같이 수정해보면 어떨까유~~?
```suggestion
  height: ${({ height }) => 
  (typeof height === 'number' ? `${height}px` : wowSpace[height]) || ''
};
```

Reply:
아래 있는 `width`도 마촨가쥐!!!

Reply:
height 가 undefined 로 들어오는 경우도 있어서 `height && height in wowSpace` 이런 식으로 undefined 가 아닌지에 대한 검증이 필요했어요! 삼항 연산자는 가독성이 안좋은 것 같아 함수로 분리해서 수정했습니당~ [e4f4c22](https://github.com/GDSC-Hongik/gdsc-client/pull/68/commits/e4f4c22aef8c3bab45d5844d27da07ddbe3cc438)

Reply:
앗 그렇군요~ 깔끔 굿굿 짱입니다용 🥰